halo/lua/lua_messaging.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+ * Vulpes (c) 2019 gbMichelle
+ *
+ * This program is free software under the GNU General Public License v3.0 or later. See LICENSE for more information.
+ */
+
+#include <lua.hpp>
+
+#include "../functions/messaging.hpp"
+
+#include "lua_helpers.hpp"
+#include "lua_messaging.hpp"
+
+static int luaV_cprint(lua_State *state) noexcept {
+    // Check if the number of arguments is correct for a print call.
+    if(lua_gettop(state) == 1) {
```

Comment:
What if the stack has more than 1 element in it? Why is 1 significant here?

Reply:
This specifically is getting the count for pushed arguments. When this function gets called by the Lua state that value will contain the number of args pushed. This is mostly in here for when I reimplement colored cprint for lua.

Reply:
I think you should add a comment explaining this